.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+## Describe the Problem and Solution
+<!-- Write 2-3 sentences summarising the reasons behind this change (this might be the problem you're solving, or the context around the request) and the solution you take. !-->  
+
+## Risk & Testing considerations
+What is the risk level of this change (Low / Medium / High)? For Medium / High please elaborate.
+
+Which modules are impacted by this change?
+
+Which integration tests and or unit tests are applicable for it? This could be a whole suite of Python tests or specific tests.
+
+Has new tests been written which exercises the relevant code paths? If yes, please identify this via the code coverage report. If not, explain why.
+
+What other testing is relevant for this change? This could include: stress testing, MongoDB patch builds, etc.
+
+## Checklist before requesting a review
+
+- [ ] I have performed a self-review of my code.
+- [ ] I have made corresponding changes to the documentation (if applicable).
+- [ ] I have added/updated tests that prove my fix is effective or that my feature works.
```

Comment:
I suggest 'demonstrate' instead of 'prove' as prove may be over-interpreted as meaning mathematically provable (https://en.wikipedia.org/wiki/Correctness_(computer_science))

Reply:
And I'd suggest adding 'correctly' after 'works'.

Reply:
Thanks for the suggestions. Updated in [8bc4113](https://github.com/wiredtiger/wiredtiger/pull/10973/commits/8bc4113ede0174b1e68e9c1b9bd909a0e0357caa)